index.html

```diff
@@ -0,0 +1,163 @@
+
+    <!--Stylesheet-->
+    <link rel="stylesheet" href="./style.css">
+
+</head>
+<body>
+    <header class="header-container">
+        <div class="nav-banner">
+            <div class="banner-icons">
+                <ul class="social-icons">
+                    <li><a href="#facebook"></a><i class="fa-brands fa-facebook-f"></i></li>
+                    <li><a href="#twitter"></a><i class="fa-brands fa-twitter"></i></li>
+                    <li><a href="#english"></a>English</li>
+                    <li><a href="#myPage"></a>My Page</li>
+                    <li><a href="#logout"></a>Logout</li>                   
+                </ul>
+            </div>
+        </div>
+        <div class="header-nav">
+            <div class="navbar">
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+                <div class="logo">
+                    <a href="index.html"><img src="./assets/images/logo.PNG" alt="merctech mentorship"></a>
+                </div>
+                <nav class="nav-menu">
+                    <ul class="menu-links">
+                      <li>
+                        <a href="about.html" class="a-links">About</a>
+                      </li>
+                      <li>
+                        <a href="#program" class="a-links">Program</a>
+                      </li>
+                      <li>
+                        <a href="#join"class="a-links">Join</a>
+                      </li>
+                      <li>
+                        <a href="#sponsor"class="a-links">Sponsor</a>
+                      </li>
+                      <li>
+                        <a href="#news"class="a-links">News</a>
+                      </li>
+                      <li class="campaign">
+                        <a href="#campaign"class="a-links">campaign</a>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+    </header>
+    <section class="hero-container">
+        <div class="container-herooo">
+            <div class="hero">
+                <div class="hero-image">
+                    <img src="./assets/images/theme11.png" class="theme" alt="theme">
+                </div>
+                
+                <div class="sub-theme">
+                    <p>We are a nonprofit organization whose goal is to build a pipeline to STEM fields (Science, Technology, Engineering, Math) for underprivileged and underrepresented groups to encourage and promote diversity.</p>
+                </div>
+                <div class="data-venue">
+                    <p class="time">2023.10.15(THU) ~ 16(FRI)</p>
+                    <p class="venue">@ National Museum of Kenya, Nairobi Centermore</p>
+                </div>
+            </div>
+        </div>
+        
+    </section>
```

Comment:
Nice section 💯 but some of its content seems to be overflowing the navbar, take a look 👇🏽 

![Screenshot 2023-03-18 at 14 21 02](https://user-images.githubusercontent.com/45739963/226126659-9ee31c99-655c-4a2e-b206-e0a0d71c066d.png)

Also look to adjust the page content a bit, this is how the page should look with the proper background and spacing 👇🏽 
![Screenshot 2023-03-18 at 14 23 51](https://user-images.githubusercontent.com/45739963/226126976-cb19ff56-7c89-4126-906d-7366d29125dd.png)

This is what you have so far 👇🏽 

![Screenshot 2023-03-18 at 14 26 09](https://user-images.githubusercontent.com/45739963/226127475-2ccebdcc-a375-4e30-ab00-1e1461a06ed0.png)

This can be fixed by adding some margins between the elements, adjusting the text sizes and having the proper background image. 🖼️